### ISSUE TITLE
ice_and_snow_on_land area type description

### ISSUE THREAD

**Issue by taylor13** [cf-convention/discuss#198]
The description in http://cfconventions.org/Data/area-type-table/10/build/area-type-table.html of the area type ``ice_and_snow_on_land`` says "The area type "ice_and_snow_on_land" means ice in glaciers, ice caps, ice sheets (grounded and floating shelves), river and lake ice, any other ice on a land surface, such as frozen flood water, and snow lying on such ice or on the land surface."  

If the description is correct, then I think the area type should be named "ice_or_snow_on_land".  The current name would imply to me that only the land area covered with snow on ice should be considered, but I think we want areas covered by snow alone or covered by ice (with snow on top or not).

**Comment by JonathanGregory** [comment]
Yes, I think you're right, Karl @taylor13, and this would be consistent with `lake_ice_or_sea_ice`. I don't think we have aliases for area types, and we cannot invalidate old data, so if we want to change `ice_and_snow_on_land` to `ice_or_snow_on_land` we might have to define the new one but keep the old one too, with a modified definition to say it's deprecated.

**Comment by taylor13** [comment]
Thanks for reviewing this.  Formally, I think the correction is consistent with language used in set theory where the union of set A and set B is described as the set comprising members of A **or** B **or** both.  The intersection of set A and set B is described as the set comprising members of both A **and** B.  Thus, ice_or_snow would be consistent with the description.  As you note we do not provide for aliases, so proceeding as you suggest seems easiest.

**Comment by davidhassell** [comment]
Hello

I agree that "or" is better than "and", here, and @JonathanGregory's suggestion sounds good.

**Comment by taylor13** [comment]
Yes, let's do it!

**Comment by JonathanGregory** [comment]
To summarise. We have agreed

* To define a new area type of `ice_or_snow_on_land` with the existing definition of `ice_and_snow_on_land`: The area type "ice_or_snow_on_land" means ice in glaciers, ice caps, ice sheets (grounded and floating shelves), river and lake ice, any other ice on a land surface, such as frozen flood water, and snow lying on such ice or on the land surface.

* To change the definition of the existing area type `ice_and_snow_on_land` to: "The area type `ice_and_snow_on_land` is deprecated. It means the same as  `ice_or_snow_on_land`, which should be used instead."

Please could the moderators @feggleton @japamment review this proposal? Thanks
